tests/views/playingScreen-test.js

```diff
@@ -104,6 +106,9 @@ describe('PlayingScreen', function () {
 
     TestUtils.Simulate.mouseOver(screen);
     expect(over).toBe(true);
+
+    TestUtils.Simulate.keyUp(screen, {key: 'Tab', which: 9, keyCode: 9});
```

Comment:
We should keep this as a separate unit test. Please refer to startScreen-tests.js for an example of how we want to structure unit tests moving forward. Unfortunately right now our tests in this project are vey inconsistent.

I suggest a test case like "should show control bar when pressing the tab key". This would include the simulated key press and then it would check that `startHideControlBarTimer` and `showControlBarByTab`